README.md

```diff
@@ -138,6 +138,10 @@ can use only `content-script` without `@extension/` prefix
 
 ## How do I disable modules I'm not using?
 
+> [!IMPORTANT]
+> Make sure you have built project before
+> `pnpm build` or `pnpm build:firefox`
```

Comment:
It appears that module-manager is likely the first command users will run right after cloning or forking the boilerplate. Requiring them to run pnpm build seems a bit unusual. Is there another solution?

Reply:
I need to move `streamFileToZip` again inside package, i've moved it to `dev-utils` to avoid deduncancy.

But if we want to do it without build, i need to move it back inside `module-manager`

Reply:
But yeah, it should work without additional build, if you don't have other idea, i need to move it back inside module.

Reply:
I've fixed it with additional `postinstall` script on `module-manager`